modules/core/src/main/java/org/apache/ignite/internal/processors/performancestatistics/PerformanceStatisticsProcessor.java

```diff
@@ -214,6 +220,27 @@ public void stopCollectStatistics() throws IgniteCheckedException {
         metastorage.write(PERF_STAT_KEY, false);
     }
 
+    /**
+     * Rotate collecting performance statistics.
+     *
+     * @throws IgniteCheckedException If rotating failed.
+     */
+    public IgniteInternalFuture<Void> rotateCollectStatistics() throws IgniteCheckedException {
+        if (!allNodesSupports(ctx.discovery().allNodes(), IgniteFeatures.PERFORMANCE_STATISTICS_ROTATE))
```

Comment:
This check is redundant. Let's remove it.

Reply:
done